lib/spack/docs/packages_yaml.rst

```diff
@@ -487,6 +487,56 @@ present. For instance with a configuration like:
 
 you will use ``mvapich2~cuda %gcc`` as an ``mpi`` provider.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Conflicts and strong preferences
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the semantic of requirements is too strong, or too verbose to exclude a single configuration
+from a set, you can also express "strong preferences" and "conflicts" from configuration files:
```

Comment:
```suggestion
If the semantic of requirements is too strong, you can also express "strong preferences" 
and "conflicts" from configuration files:
```

I think you have to know an awful lot to understand what is meant by "too verbose to exclude a single configuration from a set" here.  I think keeping it simple is better

Reply:
Done in https://github.com/spack/spack/pull/41832/commits/f02d2e84c7cde89ac397fed0da5ef6dc2aa29e06